components/thumbnail/stories/thumbnail.stories.js

```diff
@@ -127,14 +131,25 @@ export default {
 
 // @todo combine variants into one snapshot
 export const Default = Template.bind({});
-Default.args = {};
+Default.args = {
+	imageURL: "example-ava.png",
```

Comment:
Changing the default image so that it looks different from the Landscape story, this is also square so it doesn't show any space on the sides.